core/lib/store-assets.ts

```diff
@@ -13,7 +13,7 @@ const storeHash = process.env.BIGCOMMERCE_STORE_HASH ?? '';
  * @returns {string} The CDN image URL.
  */
 const cdnImageUrlBuilder = (sizeSegment: string, source: string, path: string): string => {
-  return `https://${buildConfig.get('urls').cdnUrl}/s-${storeHash}/images/stencil/${sizeSegment}/${source}/${path}`;
+  return `https://${buildConfig.get('urls').cdnUrls.at(0)}/s-${storeHash}/images/stencil/${sizeSegment}/${source}/${path}`;
```

Comment:
The point of using `at` here was so that we would handle the zero-length array case. 

If the build config guarantees a non-zero length array, I guess that's fine, although IMO it would be better to model that in the interface

Reply:
Does `source` contain the full url? If so, can we use `.find` to find the appropriate url?

Reply:
> Does `source` contain the full url? If so, can we use `.find` to find the appropriate url?

`@param {string} source - The source of the image. Can be either `content` or `image-manager`.`

Doesn't seems like?

Reply:
Ah okay.

Reply:
> The point of using `at` here was so that we would handle the zero-length array case.
> 
> If the build config guarantees a non-zero length array, I guess that's fine, although IMO it would be better to model that in the interface

Where exactly do you mean?